### DISCUSSION TITLE
Perfektes Wäremepumpen setup !

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#445] [Installation und Betrieb]
Hallo,

an der Wp. habe ich einen kontakt wenn er auf closed ist fährt die Wp die TEMPS ca. 10 grad höher, das soll sie aber nur machen wenn 100% PV anliegt. 
Ich möchte das der SHM im zeitfenster von 7:40-16:00 Uhr die WP. mit einplant und nur dann den kontakt auf closed setzt wenn 100PV% energie vom Dachen kommen. Und die WP. mit einer max Laufzeit brücksichtigt wird.

So im SAE brauche muss ich ja dazu 2 Geräte anlegen:
1 WP-Normal: Nur Zähler mit Master aktiv. Slave Schalter muß eingeschalten sein ? ohne schalter
2 WP-100% PV  min900W max3000W unterbrechnung erlaubt mit schalter
   Zähler Setup: Typ Slave Zähler, Master Zähler (WP-Normal)
   Schalter Setup: Anlaufstromerkennung (aus) Einschaltoptionen (aus)
   Schaltzeiten Tagesplan Laufzeit
   Start 7:40-16:00 Uhr
   Mindeslaufzeit 0:00  Maxlaufzeit 3:00 Stunden.

Ist das setup so richtig gemacht ?

Dankeschön 
Freundlicher Gruß

**Comment by camueller** [comment]
Passt bis auf:
- "WP-100% PV": keine Min-Leistung angeben.
- "WP-100% PV": Schalter Setup: Einschaltoption ein

**Comment by DrMabuse100** [comment]
Hallo,
Herzlichen Dank

wenn ich,
"WP-100% PV": Schalter Setup: Einschaltoption ein  mache kommt ja eine neue Maske Einschaltoption mit
Leistungschaltgrenze Dauer ein und Dauer aus. 

Was muß da genau hinterlegt werden ?

Freundlicher Gruß

**Comment by camueller** [comment]
Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/SwitchOnOption_DE.md

**Comment by DrMabuse100** [comment]
Dankeschön für den Hinweis. Wenn ich das so umsetzte:
Also Einschaltoption Aktiv.
Und Leistungsschaltgrenze 250 W  Dauer der Einschalterkennung  600s Dauer der Ausschalterkennung 60s

Habe ich aber keine Ampel mehr im Menü Status. 
Ist das so richtig ?

Freundlicher Gruß

**Comment by camueller** [comment]
Das Schalten soll ja vom SHM bei Überschuss erfolgen. Die Ampel ist diesbzgl. irrelevant und sollte ignoriert werden. Evtl. ist sie bei diesen Geräten sogar fehlerhaft.

**Comment by DrMabuse100** [comment]
Ok vielen dank für die freundlichen TIPS.

Heist das im unkehrschluss das der SHM wenn der schieber in den anteilen ob jetzt 20 Prozent Netzbebezug gewählt ist oder 0% Netzbezug 1:1 an den SAE geschickt wird. ?

Freundlicher Gruß

**Comment by camueller** [comment]
Diese Frage kann ich Dir nicht beantworten, weil das das Sunny Portal und den Sunny Home Manager betrifft. Am besten ausprobieren oder SMA fragen ...
